PROJECT_PROGRESSION.md

```diff
@@ -203,13 +173,15 @@ Impact Stage projects are expected to participate actively in CPC proceedings, a
 
 #### Acceptance Criteria
 
-To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
+To graduate from At Large stage or for a new project to join as an Impact project, a project must meet the following criteria:
 
+ * Document that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.
  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
- * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
```

Comment:
I would suggest using `AUTHORS.md` given `OWNERS.md` can be confused with `OWNERS` which is not for documentation purposes as it has an effect on how GitHub handles certain things in the repository.
```suggestion
 * Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and AUTHORS.md file showing the current and emeritus committers.
```